config/facebook_oauth.js

```diff
@@ -0,0 +1,21 @@
+dotenv = require('dotenv');
+dotenv.config();
+module.exports = {
+    development: {
+        client_id: '896021850846067',
+        client_secret: process.env.FACEBOOK_CLIENT_SECRET,
+        callback_url: 'http://localhost:4000/users/auth/facebook/callback'
+    },
+
+    test: {
+        client_id: '896021850846067',
```

Comment:
why not just take this from the env as well already? almost zero cost

Reply:
sure why not.